README.md

```diff
@@ -1,17 +1,17 @@
-## AWS IoT Jobs Client library
-*(part of the AWS IoT Device SDK for Embedded C)*
-
-AWS IoT Jobs is a service that notifies one or more connected devices of a
-pending “Job”.  A Job can be used to manage your fleet of devices, update
-firmware and security certificates on your devices, or perform administrative
-tasks such as restarting devices and performing diagnostics. For documentation
-of the service, please see the [AWS IoT Developer
-Guide](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html).
-Interactions with the Jobs service use MQTT, a lightweight
-publish-subscribe protocol.  This library provides a convenience API to
-compose and recognize the MQTT topic strings used by the Jobs service.
-The library is written in C compliant with ISO C90 and MISRA C:2012,
-and is distributed under the [MIT Open Source License](LICENSE).
+## AWS IoT Jobs library
+
+The AWS IoT Jobs library helps you notify connected IoT devices of a
+pending [Job](https://freertos.org/jobs/jobs-terminology.html). A Job
+can be used to manage your fleet of devices, update firmware and security
+certificates on your devices, or perform administrative tasks such as
+restarting devices and performing diagnostics. It interacts with the
+[AWS IoT Jobs service](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html)
+using MQTT, a lightweight publish-subscribe protocol. For documentation of
+the service, please see the [AWS IoT Developer Guide](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html).
```

Comment:
The `https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html` link has been repeated. The second link should probably point to the MQTT API here: https://docs.aws.amazon.com/iot/latest/developerguide/jobs-mqtt-api.html